### ISSUE TITLE
[QUESTION] Rewrite the code to support Termux

### ISSUE THREAD

**Issue by jcgaming-official** [aapatre/Automatic-Udemy-Course-Enroller-GET-PAID-UDEMY-COURSES-for-FREE#429]
Is it possible? It will be great to use this code on mobile devices

**Comment by AKhilRaghav0** [comment]
yes i did, use [ucf_ci](https://github.com/AKhilRaghav0/ucf_ci), will let you collect coupons code and then just redeem them.
Everything will run over github actions, so no setup required.
![image](https://user-images.githubusercontent.com/86706467/233537339-7416482b-7631-4654-9a54-8ef6c6062e13.png)


- I not promoting anything -

**Comment by KMohZaid** [comment]
looking into requirement.txt tells that this repo uses selenium, which doesn't work in android/mobile.
